ldi-orchestrator/ldio-connectors/ldio-sparql-construct/src/main/java/be/vlaanderen/informatievlaanderen/ldes/ldio/config/LdioSparqlConstructAutoConfig.java

```diff
@@ -25,12 +23,11 @@ public LdioConfigurator ldiHttpOutConfigurator() {
 
 	public static class LdioSparqlConstructConfigurator implements LdioConfigurator {
 		public static final String QUERY = "query";
-		public static final String QUERY_VALIDATION_MSG = "Must provide a valid construct query";
 		public static final String INFER = "infer";
 
 		@Override
 		public LdiComponent configure(ComponentProperties config) {
-			Query query = QueryFactory.create(Objects.requireNonNull(config.getProperty(QUERY), QUERY_VALIDATION_MSG));
```

Comment:
What happens without this nn check? Does it fail upon initialization ? Or only on the moment data is passing through the flow?

Reply:
Upon init, the ComponentProperties.getProperty() method will throw nullpointers if null